<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="windowIcon">
   <iconset theme="preferences-system" resource="images.qrc">
    <normaloff>:/icon/config</normaloff>:/icon/config</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Set File Paths For Saving and Loading</string>
     </property>
     <layout class="QVBoxLayout" name="_3">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="_8">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Default Path For Loading All Saves</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_load_path"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_set_load_path">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_6">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Path For Emulator Memory Cards  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_save_emu">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_set_save_emu">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_7">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Path For PC (.ff7) Saves</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_save_pc"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_set_save_pc">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_5">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Path For Character Stat Files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_char_stat_folder"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_set_char_stat_folder">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_4">
        <item>
         <widget class="QCheckBox" name="cb_override_def_save">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>Override Builtin New Game Data</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_default_save">
          <property name="placeholderText">
           <string>Use Only Raw Psx Files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_set_default_save">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reset_default_save_location">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Reset to Default</string>
          </property>
          <property name="icon">
           <iconset theme="edit-clear" resource="FF7tk/icons/common.qrc">
            <normaloff>:/common/edit-clear</normaloff>:/common/edit-clear</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_skip_slot_mask">
        <property name="text">
         <string notr="true">Don't update the slot mask (pc version only) </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
  <include location="FF7tk/icons/common.qrc"/>
 </resources>
 <connections/>
</ui>
